packages/sdk/src/swap/sdk.ts

```diff
@@ -161,6 +174,7 @@ export class SwapSDK {
       depositChannelId: response.id,
       depositAddress: response.depositAddress,
       brokerCommissionBps: response.brokerCommissionBps,
+      affiliateBrokers: depositAddressRequest.affiliateBrokers ?? [],
```

Comment:
I'm handling this in the swap status response
```suggestion
```

Reply:
but this is the channel creation response?

Reply:
true i was thinking it's the swap status response 👍